src/org/scratchOrgInfoApi.ts

```diff
@@ -277,6 +277,9 @@ export const requestScratchOrgCreation = async (
   scratchOrgRequest: ScratchOrgInfo,
   settings: SettingsGenerator
 ): Promise<SaveResult> => {
+  if (!hubOrg.isDevHubOrg()) {
```

Comment:
note: this method relies on the `isDevHub` prop to be set in the auth file, this prop is set based on a server check when creating and saving the auth file after this change was merged: https://github.com/forcedotcom/sfdx-core/pull/560#discussion_r853445695

if a user auth'd to a hub before that change ⬆️ was merged and starts to get the error then a re-auth should solve it.